test/Microsoft.ML.CpuMath.UnitTests.netcoreapp/UnitTests.cs

```diff
@@ -33,11 +33,11 @@ public class CpuMathUtilsUnitTests
         static CpuMathUtilsUnitTests()
         {
             // Padded array whose length is a multiple of 4
-            float[] testArray1 = new float[16] { 1.96f, -2.38f, -9.76f, 13.84f, -106.37f, -26.93f, 32.45f, 3.29f, 1.96f, -2.38f, -9.76f, 13.84f, -106.37f, -26.93f, 32.45f, 3.29f };
+            float[] testArray1 = new float[32] { 1.96f, -2.38f, -9.76f, 13.84f, -106.37f, -26.93f, 32.45f, 3.29f, 1.96f, -2.38f, -9.76f, 13.84f, -106.37f, -26.93f, 32.45f, 3.29f, 1f, 2f, 3f, 4f, 5f, 6f, 7f, 8f, 9f, 10f, 11f, 12f, 13f, 14f, 15f, 16f };
             // Unpadded array whose length is not a multiple of 4.
-            float[] testArray2 = new float[15] { 1.96f, -2.38f, -9.76f, 13.84f, -106.37f, -26.93f, 32.45f, 3.29f, 1.96f, -2.38f, -9.76f, 13.84f, -106.37f, -26.93f, 32.45f };
+            float[] testArray2 = new float[30] { 1.96f, -2.38f, -9.76f, 13.84f, -106.37f, -26.93f, 32.45f, 3.29f, 1.96f, -2.38f, -9.76f, 13.84f, -106.37f, -26.93f, 32.45f, 1f, 2f, 3f, 4f, 5f, 6f, 7f, 8f, 9f, 10f, 11f, 12f, 13f, 14f, 15f };
             _testArrays = new float[][] { testArray1, testArray2 };
-            _testIndexArray = new int[9] { 0, 2, 5, 6, 8, 11, 12, 13, 14 };
+            _testIndexArray = new int[18] { 0, 2, 5, 6, 8, 11, 12, 13, 14, 16, 18, 21, 22, 24, 26, 27, 28, 29};
```

Comment:
nit: These are all cases that will hit the "fast" (intrinsic) path.

Reply:
Yes, This will go through the intrinsics path But when we will be running these tests in avx and sse disabled mode then we will be able to test the software implementation path

Reply:
It would probably be good to have at least one code path that always hits the non-accelerated path, as it properly validates that it works in the machines default configuration (rather than only validating it works when the user explicitly disables the accelerated paths).